crates/uv-resolver/src/lock.rs

```diff
@@ -2139,6 +2139,32 @@ impl std::fmt::Display for HashParseError {
     }
 }
 
+/// Format an array so that each element is on its own line and has a trailing comma.
+///
+/// Example:
+///
+/// ```toml
+/// dependencies = [
+///     { name = "idna" },
+///     { name = "sniffio" },
+/// ]
+/// ```
+fn each_element_on_its_line_array(elements: impl Iterator<Item = InlineTable>) -> Array {
```

Comment:
Part of the decision to make here is if we want to ident single item lists and whether to indent by tabs or by spaces, cc @ibraheemdev re https://github.com/astral-sh/uv/commit/85183c1c36265f9d6942f9fb8a683c0e943243d6#diff-221a46775c0f0caff0d7be0a7669cc329bdf2c245c4b1f2400a19cc5a4f92f55R188-R194.

I believe we should use 4 spaces since it is what the ruff formatter through black and PEP does.

I'd favor indenting the item in a list item list for diff-ability. The wheels list doesn't usually change, but i'd indent it the same way for consistency.

Reply:
In comparison to our style, prettier uses two spaces and collapses short lines.

Reply:
Don't really have a strong preference here, either is fine.